src/Storage/AuthorizationCodeStorage.php

```diff
@@ -42,7 +43,41 @@ private function getUserIdByCode( $code ) {
 			return null;
 		}
 
-		return absint( $users[0]->ID );
+		if ( count( $users ) > 1 ) {
+			// This should never happen.
+			// If it does, something is wrong, so it's best to not return any user.
+			$debug_log     = "[CONCURRENTLOGINS] more than 1 user found for code: $code.";
+			$found         = 0;
+			$found_user_id = 0;
```

Comment:
I think we could replace these two variables with a single `$found_users` array.

Reply:
but then we would be collecting users and checking its length etc?

Reply:
Good point, I didn't consider that. Please ignore.